internal/common/resource/resource.go

```diff
@@ -244,6 +298,8 @@ func (a ComputeResourcesFloat) Max(b ComputeResourcesFloat) {
 	}
 }
 
+// The DeepCopy function creates a separate and distinct copy of the variable "a" and
```

Comment:
Okay. I just thought it would help with code navigation. Look at this;
![image](https://user-images.githubusercontent.com/59079323/222184941-72ace063-a637-46b3-9f9f-0a80426ea0fd.png)
By hovering around the function. I can figure out what it does, thanks to the comments